packages/components/src/components/samples/actions.ts

```diff
@@ -84,14 +85,21 @@ export function getSampleSet(config: IEntityTypeDetails): Promise<any> {
     });
 }
 
-export function updateSampleSet(config: IEntityTypeDetails): Promise<any> {
-    return new Promise((resolve, reject) => {
+export function getSampleTypeDetails(query: SchemaQuery, domainId?: number): Promise<any> {
+    return new Promise<DomainDetails>((resolve, reject) => {
+        const sampleSetConfig = {
+            domainId,
+            containerPath: ActionURL.getContainer(),
+            queryName: query.getQuery(),
+            schemaName: query.getSchema(),
+        } as Domain.GetDomainOptions;
+
         return Ajax.request({
-            url: buildURL('experiment', 'updateMaterialSourceApi.api'),
-            method: 'POST',
-            params: config,
+            url: buildURL('property', 'getDomainDetails.api'),
```

Comment:
Can this be switched to use the Domain.getDomainDetails() from @labkey/api instead of duplicating that Ajax request here?